charts/fluid/fluid/templates/role/csi/rbac.yaml

```diff
@@ -38,7 +40,10 @@ rules:
     verbs: ["get", "list", "watch", "create", "update", "patch"]
   - apiGroups: [""]
     resources: ["nodes"]
-    verbs: ["get", "list", "watch", "patch"]
+    verbs: ["get", "list", "watch", "patch", "proxy"]
+  - apiGroups: [""]
+    resources: ["nodes/log", "nodes/metrics", "nodes/proxy", "nodes/spec", "nodes/stats"]
```

Comment:
Why do we need `nodes/log`, 'nodes/metrics', 'nodes/proxy'?

Reply:
`nodes/proxy` is required to access kubelet. I will delete others.